src/libData/AccountData/AccountStoreSC.h

```diff
@@ -53,7 +53,7 @@ class AccountStoreSC : public AccountStoreBase<MAP> {
   std::mutex m_mutexUpdateAccounts;
 
   /// the blocknum while executing each txn
-  uint64_t m_curBlockNum;
+  uint64_t m_curBlockNum = 0;
```

Comment:
Is there a reason why this assignment wasn't changed to be uniform for initialization?

Reply:
Nice catch, thanks